route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -263,29 +263,36 @@ namespace route_following_plugin
     double RouteFollowingPlugin::findSpeedLimit(const lanelet::ConstLanelet& llt)
     {
         lanelet::Optional<carma_wm::TrafficRulesConstPtr> traffic_rules = wm_->getTrafficRules();
-        double target_speed;
+        double target_speed, traffic_speed =0.0, param_speed =0.0;
```

Comment:
Is there a reason to not initialize target_speed to 0 as well?

Reply:
No reason here, I can add it.